wiki/Ways-Of-Working.md

```diff
@@ -1,21 +1,104 @@
-## Tickets, Pull Requests, & the Project Board
+<!-- TODO Graham 230831: Establish a glossary of engineering terms for our wiki. -->
+_Throughout this page, "ticket" and "story" are used interchangeably to refer to GitHub issues._
 
-All work must be associated with a ticket (GitHub issue), and this ticket must have its status set to "in progress." The ticket _must_ come before the pull request, as we now use automated software which relies on this precedence.
+**Contents**
+- [Development Process Overview](#development-process-overview)
+- [Project Management Flow](#project-management-flow)
+  * [Tickets](#tickets)
+    + [Blockers](#blockers)
+    + [Story Point Estimation](#story-point-estimation)
+  * [Pull Requests](#pull-requests)
+    + [Github Quality Checks](#github-quality-checks)
+- [Standup](#standup)
+- [Deployment and QA schedule](#deployment-and-qa-schedule)
+- [Change Log](#change-log)
 
-If an engineer opens a new ticket, it can be either in-scope of a current project, or out-of-scope. 
+# Development Process Overview
 
-Engineers are welcome to open several pull requests to close an outstanding ticket. Such PRs can be either _dependent_ or _sequential_ (i.e., independent), and should be flagged accordingly, so that blocking PRs are prioritized for merging. See the [Agile Development "Story Point Estimation" section](./Agile-Development.md#story-point-estimation) for further context.
+1. We follow an Agile work philosophy. See [Agile Manifesto](https://agilemanifesto.org/) for context.
+2. We track work through GitHub's project board, using tickets' "Project" metadata. See [Project Management Flow](#project-management-flow) for full context.
+3. We use team standups to sync up on project status and share updates. Standups take place on Mondays, Tuesdays, and Thursdays at 12:00 EST. See [Standup](#standup) section for full context.
 
-All new tickets should be set to "no status" for the current cycle. Work should not begin until an engineer has been assigned to the ticket, and it has been assigned an "in progress" status.
+# Project Management Flow
+
+All work _must_ be associated with a ticket, and the ticket _must_ be opened before its respective pull request (as we use automated software which relies on this precedence).
+
+You _must not_ change the title or description of someone else's Github story or PR without permission. 
+
+## Tickets
+
+Engineers are welcome to open up new tickets. These _should_ be set to "No Status," for the current work cycle (e.g. "Cycle 6"), via the issue's "projects" metadata. Next a ticket is "teed up" (assigned the Teed Up label) by a lead or project manager.
+
+If you would like to tee-up a ticket, DM the relevant lead/PM. Do not move any stories out of our Backlog (i.e., out of the "No Status" label) yourself. 
```

Comment:
To be clear, backlog != no status. The backlog is where we place tickets handed off from product to engineering, or otherwise tickets we plan to work on but that haven't been teed up (i.e. assigned to a team member + determined as within scope for a sprint). No status, however, means there's currently no plan to address the ticket.

Reply:
Thank you! I'm going to assume, then, that line 32 applies to _both_ Backlogged and No Status labels (unless I hear different).